addons/website/views/snippets/s_website_form.xml

```diff
@@ -67,6 +67,12 @@
         </div>
 
         <!-- Field -->
+        <div data-js="WebsiteFormFieldRequired" data-selector=".s_website_form .s_website_form_model_required">
+            <!-- Disable the delete option of model required fields and show
+            alert and tooltip on delete buttons -->
+            <we-alert class="mt-1"></we-alert>
```

Comment:
@bvr-odoo I added a commit to hide the type selector for model required field. I also added this little margin top on the we-alert to make it prettier. Let me know if you think there is another thing to change :slightly_smiling_face: